packages/tools/src/account/deposit-ckb.ts

```diff
@@ -10,9 +10,9 @@ import {
   sealTransaction,
 } from "@ckb-lumos/helpers";
 import {
-  generateDepositionLock,
-  DepositionLockArgs,
-  getDepositionLockArgs,
+  generateDepositLock,
+  DepositLockArgs,
+  getDepositLockArgs,
   serializeArgs,
   getRollupTypeHash,
 } from "../modules/deposition";
```

Comment:
shall we change the file name `deposition.ts` to `deposit.ts` too?

Reply:
Updated